v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -366,10 +366,19 @@ func (f *Frontend) setupChromium() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = settings.PutIsZoomControlEnabled(false)
-	if err != nil {
-		log.Fatal(err)
+
+
+	if opts := f.frontendOptions.Windows; opts != nil  {
+		if  opts.ZoomFactor > 0.0 {
```

Comment:
Should this be `opts.ZoomFactor > 0.0 && opts.ZoomFactor != 1.0` to ignore when the default value is set?